lstchain/reco/r0_to_dl1.py

```diff
@@ -221,10 +219,30 @@ def r0_to_dl1(
     config = replace_config(standard_config, custom_config)
 
     custom_calibration = config["custom_calibration"]
-    gain_selector = load_gain_selector_from_config(config)
+
+    source_config={
+        "LSTEventSource": {
+            "allowed_tels": [1],
+            "EventTimeCalculator": {
+                "ucts_t0_dragon": int(ucts_t0_dragon),
+                "dragon_counter0": int(dragon_counter0),
+                "ucts_t0_tib": int(ucts_t0_tib),
+                "tib_counter0": int(tib_counter0)
+            },
+            "PointingSource":{
+                "drive_report_path": pointing_file_path
+            },
+            "LSTR0Corrections":{
+                "drs4_pedestal_path": pedestal_path,
+                "calibration_path": calibration_path,
+                "drs4_time_calibration_path": time_calibration_path,
+            }
+        }
+    }
```

Comment:
Why is this configuration here? Shouldn't it be placed in a config file?

Reply:
Hi @morcuended,
I used this dirty solution not to change the onsite script. 
Actually the right solution is to make this script a tool, hopefully this will happen soon.